### PR TITLE
Huge improvements

### PR DESCRIPTION
**Pull request by Dixaba**
Better startup (only one command)
Ready for CI (Azure Pipelines)
Basic changelog included

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -69,6 +69,7 @@ typings/
 
 # production
 /build
+/dist
```
Comment (Dixaba) [inline .gitignore:72]: Ignore distribution artifacts

File: README.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # TurboAery
+[![Build Status](https://dev.azure.com/dixaba/Dixaba/_apis/build/status/Dixaba.TurboAery?branchName=master)](https://dev.azure.com/dixaba/Dixaba/_build/latest?definitionId=14&branchName=master)
```
Comment (Dixaba) [inline README.md:2]: Build badge

File: package.json [hunk 3]
```diff
@@ -2,14 +2,33 @@
   "name": "turboaery",
   "version": "0.0.1",
   "description": "Assistant tool for League players",
-  "main": "public/Main.js",
+  "main": "public/electron.js",
```
Comment (Dixaba) [inline package.json:5]: ```electron-builder``` requires this name for start script

File: package.json [hunk 4]
```diff
@@ -2,14 +2,33 @@
...
+      "target": "zip"
+    },
+    "mac": {
+      "target": "zip"
+    },
+    "directories": {
+      "buildResources": "public"
+    }
+  },
+  "homepage": "./",
```
Comment (Dixaba) [inline package.json:31]: To build ```react``` with relative path

**Review by arsenykruglikov — APPROVED** [review]
It's ok now

**Pull request merged**